src/components/Slider/SliderHelpers.js

```diff
@@ -0,0 +1,52 @@
+export function calcDimensions({ isRange, max, min, value }) {
+  if (!isRange) {
```

Comment:
why not convert it to hook and perform this calculations only when one of the arguments are changes?

Reply:
or at least create a hook that using this function instead of using it directly

Reply:
This approach is more performant and flexible - it pure function - easy for reuse, easy for change easy for tests.
We already have all needed variables from hooks - so I thought that simple helper is better for use.